src/lab13.py

```diff
@@ -1339,14 +1339,15 @@ def composite(self):
                 max(self.active_tab_height,
                     layer.absolute_bounds().bottom())
 
-    def clone_latest(self, visual_effect, current_effect):
-        node = visual_effect.node
+    def clone_latest(self, parent_effect, child_effect):
```

Comment:
Should we maybe name it `old_parent_effect`? Willing to merge as is too.

Reply:
I like parent_effect. "old" people might think of as "from previous frame"